### ISSUE TITLE
(e || []).filter is not a function when running envinfo

### ISSUE THREAD

**Issue by henrycity** [tabrindle/envinfo#187]
When I tried to use envinfo either through `npm i -g envinfo` or `npx envinfo` on `macOS 10.15.6`, I had the same issue. The version of envinfo I just installed is `7.7.1`. I also tried to use different node versions and I still couldn't run (`14.8.0`,  `12.18.3`, `10.22.0`)

```
(node:17815) UnhandledPromiseRejectionWarning: TypeError: (e || []).filter is not a function
    at /Users/henrycity/Projects/posti-web-components/node_modules/envinfo/dist/envinfo.js:1:73314
    at async Promise.all (index 72)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:17815) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:17815) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by unadlib** [comment]
#184 some issue

**Comment by tabrindle** [comment]
closed via #185

**Comment by tabrindle** [comment]
fix in 7.7.3
